src/classes/PMT_PaymentWizard_CTRL.cls

```diff
@@ -154,43 +154,55 @@ public with sharing class PMT_PaymentWizard_CTRL {
 
     /*******************************************************************************************************
     * @description decimal for amount already written off
-    */ 
+    */
     private Decimal amountWrittenOff { get; set; }
 
 
     /*******************************************************************************************************
     * @description Integer for count of invalid payment records
-    */ 
+    */
     private Integer invalidPaymentRecordCount { get; set; }
 
     /*******************************************************************************************************
     * @description Boolean to indicate if the wizard page should be enabled
-    */ 
+    */
     public Boolean isWizardEnabled { get; set; }
-    
+
     /*******************************************************************************************************
     * @description gets the payment related settings
     * @return npe01__Contacts_and_Orgs_Settings__c the settings object
-    */ 
+    */
     public npe01__Contacts_and_Orgs_Settings__c getPaymentsSettings(){
         if (paymentsSettings == null) {
            paymentsSettings = UTIL_CustomSettingsFacade.getContactsSettings();
         }
-        
+
         return paymentsSettings;
     }
-  
+
+    /*******************************************************************************************************
+    * @description Returns true if Payments enabled is true on paymentsSettings
+    * @return Boolean
+    */
+    public Boolean getIsPaymentsEnabled() {
```

Comment:
This will definitely work, but I think you should change the `paymentsSettings` var to private since it no longer needs to be exposed on the VF page.

Reply:
@force2b , When I change this var to private, then when logging as standard user, I get the method not visible error when I navigate to schedule payment page.

Reply:
![Screen Shot 2020-01-10 at 4 08 32 PM](https://user-images.githubusercontent.com/52580237/72190038-22486d80-33c4-11ea-85ab-a5d471b5d0ab.png)

Reply:
Did you change this method to private? If so, what I was recommending above was to change the `paymentSettings` var to private.

Reply:
> Did you change this method to private? If so, what I was recommending above was to change the `paymentSettings` var to private.

Yes, I got it wrong, sorry, will change paymentsSettings, @force2b